extras/cronwrapper

```diff
@@ -59,12 +59,9 @@ else
 	RET=$?
 fi
 
-if [ $RET -ne 2 -a $RET -ne 5 ]; then
+if [ $RET -ne 0 ]; then
```

Comment:
I **think** this is the right move, but I'm a little bit conflicted (mostly since this isn't how I use plexupdate). If you're only using it to notify you when an update is available, this would do that. Daily. Until you finally installed it. Which, to me, makes it sound really annoying. :)

BUT, if the reason you're running this is to make sure you're notified when there's a newer version than the one you have installed available, then this is exactly what it's doing, so...